### ISSUE TITLE
Send direct messages (DM) to new followers

### ISSUE THREAD

**Issue by httpmerak** [InstaPy/InstaPy#6444]
## Expected Behavior
Hello good afternoon

A few years ago (between 2017 and 2020) there were some requests to send direct messages (DM) to new followers; these requests were refused on the grounds that Instagram Web does not offer such a service, however, today it is possible to send direct messages via the computer and this possibility would be of great value.

In case there is a spelling error, I'm sorry, English is not my mother tongue and my knowledge of it is mediocre.
See you later...

## Current Behavior
Nonexistent

**Comment by CodeMasterBond** [comment]
I do not believe sending direct messages off instapy was ever made a thing. I know you were able to do it using other python scripts however instapy doesn’t have that made a function yet. You are always more than welcome to create a pull request with a dm function but if not, that’s totally cool too. And your English is very good :)

**Comment by MER4K** [comment]
Hi my friend, @CodeMasterBond, coldly, thanks for the feedback.
I understand that InstaPy does not yet have a function implemented for sending straight messages.

I can't guarantee much about creation on my part, but I'll try to develop something, do you have any support material you can provide on the paths I should follow?

My programming knowledge is relatively low and when it comes to Python even less.

See you soon!

**Comment by CodeMasterBond** [comment]
As of now, I am not aware of any open creation paths. I understand the function of dm's would be convenient for increasing followers, although the function is yet to be made. Also, if you are interested in learning python, I recommend using w3schools as a resource. I myself have learned a lot from its programs, and its free. ``https://www.w3schools.com/python/default.asp`` There is the link for the python course. Thanks!

**Comment by MER4K** [comment]
Perfect, I'm reviewing some more basic courses on programming and will be moving to Python soon.

I'll definitely take a look at the link you sent me, thank you very much.

I couldn't forget, a happy New Year for you and everyone who is important, that this year is amazing.

See you soon!

**Comment by CodeMasterBond** [comment]
Thank you for the wishing!

 Happy New Year to you too!!! 🎉

**Comment by hunterverhelst** [comment]
Hello! Can I work on this issue?

**Comment by CodeMasterBond** [comment]
Hey @hunterverhelst, wdym “work on this issue?” If you would like to make a pull request for making DM a function, then go ahead. You are more than welcome to do it.

**Comment by loic-hamdi** [comment]
@hunterverhelst  thanks for the PR, too bad it is still not accepted after more than 2 years.
Do you know if this is still work in production?
